docs/data/joy/components/textarea/textarea.md

```diff
@@ -115,6 +115,20 @@ It's usually more common to see textarea components using decorators at the top
 
 {{"demo": "TextareaDecorators.js"}}
 
+### Textarea ref
+`useRef` in React is typically used to reference a DOM element or a component. In the case of form inputs like Input or TextArea in Joy UI, you can use useRef to access and interact with the underlying DOM node directly without causing re-renders.
+
+{{"demo": "TextareauseRef.js"}}
```

Comment:
```suggestion
{{"demo": "TextareaRef.js"}}
```

You have to create a demo file named `TextareaRef.js` too.

Reply:
I have created demo file named TextareaRef.js and also made changes which you suggested. Thank you for reviewing .